backends/xnnpack/third-party/xnnpack.buck.bzl

```diff
@@ -1,38 +1,47 @@
 load("//third-party:glob_defs.bzl", "subdir_glob")
 load(
     ":xnnpack_src_defs.bzl",
-    "HOT_SRCS",
```

Comment:
@digantdesai I also did a lot of clean up on this buck.bzl file, do you mind taking a look at this one as well.